client/rest-client/src/main/java/org/infinispan/client/rest/configuration/SslConfiguration.java

```diff
@@ -107,6 +111,10 @@ public String protocol() {
       return protocol;
    }
 
+   public String provider() {
```

Comment:
didn't find any usage for this method except `org.infinispan.client.rest.configuration.SslConfigurationBuilder#read()`.
Is the configured provider used anywhere?

Reply:
Fixed